File: SampleSetup/Output/TypeScript/SampleProject/Protocol/Model/ApiCountry.ts
import { AbstractModelWithId } from './AbstractModelWithId';
import { ApiCity } from './ApiCity';
import { ApiAddress } from './ApiAddress';
export class ApiCountry extends AbstractModelWithId {
    TitleShort: string;

    TitleLong: string;

    PhoneCode: string;

    Code: string;

    Cities: Array<ApiCity>;

    Addresses: Array<ApiAddress>;

}

File: SampleSetup/Output/Kotlin/SampleProject/Protocol/Model/ApiCountry.kt
package SampleProject.Protocol.Model

import SampleProject.Protocol.Model.*
open class ApiCountry: AbstractModelWithId() {
    var TitleShort: String

    var TitleLong: String

    var PhoneCode: String

    var Code: String

    var Cities: ArrayList<ApiCity> = ArrayList<ApiCity>()

    var Addresses: ArrayList<ApiAddress> = ArrayList<ApiAddress>()

}
